### ISSUE TITLE
Figure out how to make use of sample data from NCBI

### ISSUE THREAD

**Issue by jaeddy** [mc2-center/csbc-pson-dcc#56]
@bswhite, @vpchung, @andrewelamb - This may have gotten lost after I closed the related issue, but I've debugged my NCBI crawling code and now have a [**table of metadata**](https://www.synapse.org/#!Synapse:syn22015979/tables/) for 14,196 samples (PubMed ID -> BioProject ID -> BioSample in GEO, SRA, maybe others). At a minimum, we can probably use this to make sure we're not missing any datasets or files as link entities. But we might be able to fill some gaps in annotations as well.

Note: I should check out the code in #43 to see if the table can be further augmented.

**Comment by jaeddy** [comment]
@bswhite, @vpchung, @jaybee84 - I've added another table ([**`bioproject_combined`**](https://www.synapse.org/#!Synapse:syn22041415/tables/)) that merges BioProject metadata with all the study/sample/experiment/run metadata from SRA and GEO. 

I don't think the process for collecting the information — nor the content or structure of the data itself — is mature enough to use for any sort of automation. However, I hope it can serve as a tool/resource to facilitate the manual dataset annotation.

As you go, you should be able to plug a BioProject ID into the query for `project_acc` and take a look at all the related information — e.g.,

```
SELECT * FROM syn22041415 WHERE ( ( "project_acc" = 'PRJNA388108' ) )
```

(I've also added the column as a facet)

Let me know if you have any questions about column names, sources, etc.

The info might also be relevant when we get to annotating files... but I'm trying not to think about that yet. 🙂

**Comment by bswhite** [comment]
Thanks, @jaeddy . I passed on to Jineta and Verena a simple wrapper around this table to create the externalLink markup given a BioProject PRJNA id. The guts of this is selecting the GSE (study_center_project_name) and SRA (study_accession) ids like:

`SELECT DISTINCT project_acc,study_accession,study_center_project_name FROM syn22041415 where project_acc = 'PRJNA476340'`

In this example, both the GSE and SRA ids are NA. I manually looked up all ~15-20 of these failed cases and in each case there was no SRA associated with the GSE. Is this table built starting at SRA ids? There is a GSE id associated with this BioProject id:
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115870

Just a warning that this table may not be completed -- but still helpful in filling in the frequent case where there is a GSE and associated SRA.
